test/com/google/enterprise/adaptor/database/JdbcFixture.java

```diff
@@ -88,9 +88,11 @@
           dbuser = properties.getProperty("test.jdbc.user");
           dbpassword = properties.getProperty("test.jdbc.password");
         }
-      } catch (FileNotFoundException | IllegalArgumentException e) {
+      } catch (NoSuchFileException | IllegalArgumentException e) {
```

Comment:
Why? Just why did they need to do this? Surely somebody must have asked "Why do you need a new exception for this, why not just throw FileNotFoundException?"